quickwit-cli/src/cli.yaml

```diff
@@ -0,0 +1,107 @@
+name: Quickwit CLI
+about: Index and search structured or unstructured data from the command line
+author: Quickwit, Inc. <hello@quickwit.com>
+
+subcommands:
+    - new:
+        about: Creates an index
+        args:
+            - index-path:
```

Comment:
index-uri?

Reply:
In find that "uri" is not very user-friendly so I'd rather use "path" externally and "uri" internally. WDYT?